src/patterns/start-pages/index.md.njk

```diff
@@ -1,41 +1,72 @@
 
-{{ example({group: "patterns", item: "start-pages", example: "default", html: true, nunjucks: true, open: false}) }}
+The public-facing version of your service will need to start on a GOV.UK content page, published using one of the GOV.UK publishing apps. You’ll usually need to agree content for this GOV.UK content page with the GDS content team. Read [guidance on how to get your service on GOV.UK](https://www.gov.uk/service-manual/service-assessments/get-your-service-on-govuk).
 
-## When to use this pattern
+## How it works
 
-If your service is in public beta or live, you must have a starting point hosted on GOV.UK. This will link your service to the rest of GOV.UK.
+A service start point should:
+- give the user just enough information to help them understand what the service does and whether it will meet their need - including [giving the service an name that reflects the problem it solves for users](https://www.gov.uk/service-manual/design/naming-your-service)
+- include a ‘button’ linking into the service, with text that’s consistent with the action you’re asking users to take - for example, ‘Start now’, ‘Sign in’ or ‘Register or update your details’ (if you need a secondary call to action, use a standard link)
+- let users sign in, resume an application they’ve already started or update their details (if relevant)
+- include any other information that most users are likely to need before they start using the service online - for example, how much it costs to use the service and roughly how long it will take
+- include details of other ways to access the service — for example, by telephone or by completing a paper form
 
-You can use the coded example here to test what works best for your users in prototypes.
+You’ll also need to list any documents or information the user will need to help them complete the service (there’s no need to list information that a user is likely to know from memory — for example, their own date of birth).
 
-## How it works
+You can list the documents or information at the start point, if it doesn’t make the page too complicated. Or you can move these details to a separate ‘What you’ll need’ page, served within the service subdomain.
+
+## Eligibility
+
+Avoid using the start point to provide complex eligibility information — instead, ask questions inside the service to establish whether the user is eligible.
+
+Read [guidance on designing how GOV.UK content and transactions work together](https://www.gov.uk/service-manual/design/govuk-content-transactions).
+
+## Choosing a format for a start point on GOV.UK
+
+The options available depend on whether the start point will be published as ‘mainstream’ or ‘Whitehall’ content.
+
+The content team at GDS is responsible for creating mainstream start points (as with other mainstream content). ‘Mainstream’ content is content aimed at the general public — or at meeting business needs that aren’t sector-specific.
```

Comment:
Putting the second sentence first might help users who are curious about what 'mainstream' means. Do we need to capitalise 'content team' (because it's a GOV.UK team name)?

```suggestion
‘Mainstream’ content is aimed at the general public, or at meeting business needs that are not sector-specific. As with other mainstream content, the content team at GDS is responsible for creating mainstream start points.
```

Reply:
Looks like lowercase is fine (as content isn't a product name).

Given the 'government teams' audience for this page, the key info is "the content team." 

To highlight the explanation a bit better we could separate the second sentence into a new paragraph?

Reply:
@calvin-lau-sig7 Sounds good!